multigeiger/multigeiger.ino

```diff
@@ -419,13 +417,12 @@ void setup() {
   // just for fun
   log(INFO, "Let's go!");
 
-  // build revString
-  sprintf(revString, "V%d.%d.%d %s", VERSION_MAJOR, VERSION_MINOR, VERSION_PATCH, VERSION_DATE);
```

Comment:
will now only be the full VERSION_STR (including "-dev" for development versions).